apps/utils/compute_groundtruth.cpp

```diff
@@ -26,8 +26,10 @@
 #else
 #include <stdlib.h>
 #endif
+
 #include "filter_utils.h"
 #include "utils.h"
+#include "../../src/math_utils.cpp"
```

Comment:
can you include #include "math_utils.h"?

Reply:
No, it doesn't know the path.